### DISCUSSION TITLE
Vite: Output Ressource URLs as absolute path in development?

### DISCUSSION THREAD

**Discussion by svenhoerig** [adonisjs/core#4619] [Help]
In my project (AdonisJS 6 + Vite) I have the situation that the resource URLs have to be output as absolute URLs. The resources are integrated into the HTML using `@vite(['resources/css/app.css', 'resources/js/app.js'])`. But all my attempts have failed, the CSS and JS files are always, at least in development mode, only integrated as relative and not absolute URLs.
Is there a way to change this?

<img width="549" alt="Bildschirmfoto 2024-06-16 um 18 42 09" src="https://github.com/adonisjs/core/assets/32217846/7a14d535-e81d-4a48-aa3d-146df2d0f405">

**Comment by RomainLanz** [comment]
Hey @svenhoerig! 👋🏻 

Could you please give us your use case that needs to have those resources as absolute URLs?

**Reply by svenhoerig** [reply]
Hi @RomainLanz , 
my use case is, that I have an edge Template which I use to generate a PDF file. In the PDF file the URLs have to be absolute URLs, thats why my resource URLs have to be outputted as absolute URLs.

**Reply by svenhoerig** [reply]
@RomainLanz any news on this?

**Reply by RomainLanz** [reply]
Yes, sorry for the delay.

It is not currently doable, but it will be a great addition.
May you create an issue in https://github.com/adonisjs/vite.

**Reply by svenhoerig** [reply]
Created Issue: https://github.com/adonisjs/vite/issues/17
